int32[] theta_index
float32[] est_rho
float32[] est_theta
float32[] delta_rho
geometry_msgs/Polygon[] endpoints
geometry_msgs/Polygon[] lengths
geometry_msgs/Polygon[] endpoint_ranges
geometry_msgs/Polygon[] theta
